src/assets/svg/resources-contact.svg

```diff
@@ -0,0 +1 @@
+<svg fill="none" height="41" viewBox="0 0 41 41" width="41" xmlns="http://www.w3.org/2000/svg"><path d="m38.75 8.75h-4v18h-26v4c0 1.1.9 2 2 2h22l8 8v-30c0-1.1-.9-2-2-2zm-8 12v-18c0-1.1-.9-2-2-2h-26c-1.1 0-2 .9-2 2v28l8-8h20c1.1 0 2-.9 2-2z" fill="#03c7d3"/></svg>
```

Comment:
for what it's worth, I've been trying to give all our SVGs CSS classes (like fillStd) instead of hard-coded colors. It makes it much easier to do hover effects and align them with our color scale.

Reply:
Thanks for catching this! I agree with you and this was unintentional.